gapis/replay/replay.go

```diff
@@ -59,3 +59,13 @@ type Config interface{}
 // to insert a postback of the currently bound render-target content at a
 // specific atom.
 type Request interface{}
+
+// Result is the function called for the result of a request.
+// One of val and err must be nil.
+type Result func(val interface{}, err error)
+
+// RequestResult is a pair of Request and Result.
+type RequestResult struct {
```

Comment:
Nit:  Call it RequestResultPair or RequestAndResult ? 
I though for a while it is "the result of request"